### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by ghost**
Update the dockerfile generated by Visual Studio so that it actually works.

### CODE REVIEW

**Comment by arglanc** [comment]
Closes #31

**Comment by arglanc** [comment]
Fixes #31

File: Gatekeeper/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
-RUN dotnet build "Gatekeeper.csproj" -c Release -o /app
+COPY *.csproj ./
+RUN dotnet restore
 
-FROM build AS publish
-RUN dotnet publish "Gatekeeper.csproj" -c Release -o /app
+COPY . ./
+RUN dotnet tool install -g Microsoft.Web.LibraryManager.Cli
+RUN /root/.dotnet/tools/libman restore
+RUN dotnet publish -c Release -o out
```
Comment (DavidFair) [inline Gatekeeper/Dockerfile]: Does this build the release copy in the docker container then? If so this should be built on Travis, then the artifacts copied across.

Comment (arglanc) [reply Gatekeeper/Dockerfile]: They're built in the build-env container which is thrown away once the image is created.  The only stuff in the final image is from line 12 onwards, where it copies the compiled output from build-env.

Comment (DavidFair) [reply Gatekeeper/Dockerfile]: 👍

**Review by DavidFair — APPROVED** [review]

**Pull request merged**